boards/arm/nrf51dk_nrf51422/doc/index.rst

```diff
@@ -1,13 +1,13 @@
-.. _nrf51_pca10028:
+.. _nrf51dk_nrf51422:
 
-nRF51-PCA10028
-##############
+nRF51 DK
+########
 
 Overview
 ********
 
-The nRF51 Development Kit (PCA10028) hardware provides support for the Nordic
-Semiconductor nRF51822 ARM Cortex-M0 CPU and the following devices:
+The nRF51 Development Kit (nRF51 DK) hardware provides support for the Nordic
+Semiconductor nRF51422 ARM Cortex-M0 CPU and the following devices:
```

Comment:
It would be nice if these renames retained the association of the old identifier in at minimum the documentation, and possibly other places.  Something as simple as "In earlier Zephyr releases this board was known as `nrf51_pca10028`".  Some people will only know these boards by this name, not the marketing DK name.

This is useful even if we have perfect toolchain infrastructure to allow continued use of the existing name.

Reply:
Yes, I think we could add a sentence here as you propose.

Reply:
Will add a separate commit for that, to be reviewed.

Reply:
Added a note @pabigot .